core/storage/statistics/gae_models.py

```diff
@@ -1134,6 +1136,224 @@ def delete_playthroughs_multi(cls, playthrough_ids):
         cls.delete_multi(instances)
 
 
+class LearnerAnswerDetailsModel(base_models.BaseModel):
+    """Model for storing the answer details that a learner enters when they
+    are asked for it, per state one model instance is created. And currently
```

Comment:
... are asked for an explanation of their answer. Currently, the model supports exploration states and questions. One instance of this model is created for each of these objects.

The ID of this model is in the format XXX.  <-- you need to specify this too, it's important

Reply:
Done